packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1280,6 +1295,72 @@ export default class Meetings extends WebexPlugin {
       });
   }
 
+  /**
+   * sort out locus array for initial creating
+   * @param {Array} loci original locus array
+   * @returns {undefined}
+   * @public
+   * @memberof Meetings
+   */
+  sortLocusArrayToUpdate(loci: any) {
```

Comment:
issue: presumably the type here is at least of an array i.e. not any but at least `any[]`